docs/modules/ROOT/pages/getting-started.adoc

```diff
@@ -81,33 +83,36 @@ Check that the API is accessible:
 
 [source,shell]
 ----
+echo http://lieutenant.${K3S_INGRESS_IP}.nip.io/healthz
 curl http://lieutenant.${K3S_INGRESS_IP}.nip.io/healthz
 ----
 This should return `ok` as the answer to the `curl` command. You can see this is the same API Commodore and Steward will use (in this getting started guide without https and using the nip.io dynamic URL).
 
 TIP: The API documentation can be accessed in your browser under http://lieutenant.${K3S_INGRESS_IP}.nip.io/docs.
 
-=== Prepare Lieutenant Operator access to GitLab.com
+=== Prepare Lieutenant Operator access to GitLab
 
-Lieutenant needs API access to a Gitlab server. This is required to create and manage Git repositories for clusters and tenants.
+Lieutenant needs API access to a GitLab server. This is required to create and manage Git repositories for clusters and tenants.
 
 [NOTE]
 .What are tenants?
 ====
-A "tenant" is just an entity to assign clusters to. It could be a customer, a department, a team, or anything you want to group clusters with. This concept is also used by Commodore, as every tenant gets his own configuration Git repository to (for example) apply common settings to all clusters belonging to a particular tenant. Any cluster specific configuration values are stored in that tenant's own configuration Git repository.
+A "tenant" is an entity to assign clusters to. This entity could be a customer, a department, a team, or anything you want to group clusters with. This concept is also used by Commodore, as every tenant gets his own configuration Git repository to (for example) apply common settings to all clusters belonging to a particular tenant. Any cluster specific configuration values are stored in that tenant's own configuration Git repository.
 ====
 
-Create a Kubernetes secret which contains the access token for the GitLab.com API, which can be generated here: https://gitlab.com/profile/personal_access_tokens (needs `api` scope).
+Create a Kubernetes secret which contains the access token for the GitLab API, which can be generated here: https://gitlab.com/profile/personal_access_tokens (needs `api` scope, amend gitlab.com with your own GitLab instance URL if needed).
 
-Replace `MYTOKEN` with the generated GitLab.com API token.
+Replace `MYTOKEN` with the generated GitLab API token. If you're using your own GitLab instance, amend `GITLAB_ENDPOINT` and `GITLAB_SSH_HOST_KEY` (You can use the `ssh-keyscan` tool to get the SSH host key of your own GitLab instance).
 
 [source,shell]
 ----
-export GITLABCOM_TOKEN=MYTOKEN
+export GITLAB_TOKEN=MYTOKEN
+export GITLAB_ENDPOINT=gitlab.com
+export GITLAB_SSH_HOST_KEY="gitlab.com ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIAfuCHKVTjquxvt6CM6tdG4SLp1Btn/nOeHHE5UOzRdf"
```

Comment:
As mentioned before; I fear Argo CD requires RSA host keys to function properly (not 100% sure though)

Reply:
Yeah, this will be fixed in #64